### ISSUE TITLE
Do not pollute site-packages

### ISSUE THREAD

**Issue by moubctez** [PyCQA/pylint-plugin-utils#29]
Currently, `LICENSE` and `README.md` get installed into _site-packages_ directory. This is bad, for obvious reasons.

I used this patch to fix the problem:

```
--- pyproject.toml.orig	2023-05-19 20:37:05.000000000 +0000
+++ pyproject.toml
@@ -24,11 +24,6 @@ keywords=["pylint","plugin","helpers"]
 packages = [
   {include = "pylint_plugin_utils/"}
 ]
-include = [
-  "LICENSE",
-  "*.rst",
-  "*.md"
-]
 
 [tool.poetry.dependencies]
 python = ">=3.7,<4.0"
```

**Comment by carlio** [comment]
Hmm, this appears to be an unintended side effect of using poetry that I did not know about: https://github.com/python-poetry/poetry/issues/2015#issuecomment-1133467892

I will figure out how to fix this properly. Removing the `include` section does prevent the pollution, though if possible I would like to include the LICENSE file.

**Comment by carlio** [comment]
I have fixed this and released `0.8.2` now. The LICENSE is available inside the `.dist-info` directory along side the source code, which is sufficient in my view, and it will not be accidentally installed in the `site-packages` directory now.



(ignore the failing build, coveralls.io API just failed)

**Comment by carlio** [comment]
Note that this same issue will be in other libraries I maintain so likely the LICENSE file will still be there just from another library, if you're using something like pylint-django or prospector... I will go fix those now too.
